### PR TITLE
Reintegrate puppeteer tests

### PR DESCRIPTION
**Pull request by cvalarida**
## Description
This PR re-enables puppeteer tests running in CI. The idea here is that while they may still be flaky, they're running in debug mode now and should give a better sense of _why_ they're failing.

## Testing done
That's what this is PR is for? :man_shrugging: 

## Screenshots
N/A

## Acceptance criteria
- [x] The tests run in CI

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -67,8 +67,7 @@ node('vetsgov-general-purpose') {
         parallel (
           e2e: {
             sh "export IMAGE_TAG=${commonStages.IMAGE_TAG} && docker-compose -p e2e up -d && docker-compose -p e2e run --rm --entrypoint=npm -e BABEL_ENV=test -e BUILDTYPE=vagovprod vets-website --no-color run nightwatch:docker"
-            // Temporarily disabling puppeteer tests due to flakiness
-            // sh "docker-compose -p e2e run --rm --entrypoint npm -e BABEL_ENV=test -e BUILDTYPE=vagovprod vets-website --no-color run test:puppeteer:docker"
+            sh "docker-compose -p e2e run --rm --entrypoint npm -e BABEL_ENV=test -e BUILDTYPE=vagovprod vets-website --no-color run test:puppeteer:docker"
```
Comment (ncksllvn) [inline Jenkinsfile]: I think we could wrap this in a [try/catch](https://github.com/michaelrice/groovy_example_scripts/blob/master/try-catch-example.groovy) and not worry too much about disrupting other engineers.

**Review by ncksllvn — APPROVED** [review]
LGTM

**Comment by cvalarida** [comment]
Ooh, I like those ideas. I think for now, I'll leave it like it is so I can start getting data sooner rather than later, but I've set up a slackbot reminder to loop back around to this. Thanks!

**Comment by cvalarida** [comment]
Okay, so I actually _did_ end up wrapping the puppeteer test line in a `try / catch` to notify me when it fails so I can investigate without blocking other PRs. Thanks for the idea, @ncksllvn! :raised_hands: 

I'm thinking I'll leave changing the environment variable to a parameter for later, though.

**Review by jbalboni — APPROVED** [review]

**Review by bshyong — APPROVED** [review]

**Pull request merged**